app/containers/scenes/ProfileScene.js

```diff
@@ -1,24 +1,90 @@
 import React, { Component } from 'react';
 import {
   View,
-  Text
+  Text,
+  TextInput,
+  Image,
+  TouchableHighlight
 } from 'react-native';
 
+import { ProfileSceneCss as styles } from './ProfileScene.css.js'
+
 export default class ProfileScene extends Component {
 
   constructor(props) {
-    super(props);
+    super(props)
+    let data = JSON.parse('{"id":"0a9ds7a767ac7vr5as4de4465464646","firstName":"Tina","lastName":"Turner","displayName":"Tina Turner","email":"tina@singer.com","birthday":"Aug 20, 1998","interestedChallenges":["d53h338383hjsjbs623yjb923u2h2939","9a7a5d5d7v8a7hwh11199dhsdjb2387y"],"completedChallenges":["6a6a5a55sdfh12j3jb1423123412jhkjs"],"createdChallenges":["6a6a5a55sdfh12j3jb1423123412jhkjs"],"created":"Mon Feb 06 2017 00:45:48 GMT-0700 (MST)"}')
+    this.state = {
+      userData: data,
+      completedCount: data.completedChallenges.length,
+      createdCount: data.createdChallenges.length,
+      profChar: data.firstName.charAt(0),
+      edit: false
+    }
   }
 
-  goBack() {
-    this.props.navigator.pop();
+  // TODO: load profile image into circle,
+  // TODO: have default but maybe customizable image behind
+
+  editDisplayName() {
+    this.setState(prevState => ({
+      edit: this.state.edit ? false : true
+    }))
   }
 
+  endEdit() {
+  }
+
+  // if the name goes unchanged it will be null
+  submitNameChange(name) {
+    if(name != undefined) {
+      // change the name
+    }
+  }
   render() {
     return (
-      <View>
-        <Text style={{textAlign:"center"}}>[Profile Scene]</Text>
+      <View style={{flex: 1}}>
+        <View style={styles.bottom}>
+          <View>
+            <Image
+            style={styles.backgroundImage}
+            source={{uri:'https://firebasestorage.googleapis.com/v0/b/misviajecitos-3d2e6.appspot.com/o/capitolreef%2FUNADJUSTEDNONRAW_thumb_2a6.jpg?alt=media&token=695339ef-25ff-49d7-999f-80648ca37fcd'}}/>
```

Comment:
I think we need to have this stored locally in the app. If they are offline, this will break. Also, we need to make sure we have rights to use this image. [Pexels](https://www.pexels.com/) is a website where you can find images that have open licenses. Maybe check it out.

Another thought, I'm not sure how we are getting profile and cover photos, or if they are even available. Originally I envisioned using those, but I don't know if we can get them. I just thought I'd bring it up.